### ISSUE TITLE
Remove requirement for cards to have both term and definition filled in

### ISSUE THREAD

**Issue by ahmadshariq711** [hwgilbert16/scholarsome#153]
When creating flashcards, and both term and definition is empty, the save button doesn't work. A red alert shows up, on the flashcard with the issue, but it would be better if the alert also shows up on top near the save button to increase user-friendliness.

![image](https://github.com/hwgilbert16/scholarsome/assets/86062499/749f43b6-4bd8-4b44-948e-681323444356)
This is the empty flashcard before saving
![image](https://github.com/hwgilbert16/scholarsome/assets/86062499/830f7fbe-e35a-4df0-aef0-7d040138e0e0)
This is after clicking the save button
![image](https://github.com/hwgilbert16/scholarsome/assets/86062499/c737ac8d-733a-49d1-a00d-e9ec145501ee)
It would be better to also have an alert message near the save button as the flashcard that is empty may not be visible

**Comment by hwgilbert16** [comment]
I'll see if I can get the page to scroll to the first card with missing fields. If not, an alert up the top can be added.

**Comment by ahmadshariq711** [comment]
Thank you!

**Comment by BrianENguyen** [comment]
If possible, it would be a lot better if you removed the requirement to have non-blank cards in a study set. It gets very tedious when saving a set, only to find out that there's a blank card. This would make me continuously scroll the page until I find it. Quizlet doesn't have non-blank cards requirement

**Comment by hwgilbert16** [comment]
> If possible, it would be a lot better if you removed the requirement to have non-blank cards in a study set. It gets very tedious when saving a set, only to find out that there's a blank card. This would make me continuously scroll the page until I find it. Quizlet doesn't have non-blank cards requirement

Good point. Initially, I was concerned about blank cards being a potential DOS attack vector. However, with the current ratelimiting that is in place, this is not a possibility.

This requirement will be removed in the next release.
